### ISSUE TITLE
GL surface dimensions are not updated sometimes

### ISSUE THREAD

**Issue by sfan5** [mpv-android/mpv-android#82]
@jeeb could reproduce this on switching from mpv to any app and back
on my device I could only reproduce this by switching from mpv to a <i>newly opened</i> app (via a notification) and then back

the core issue is that `android_reconfig` (mpv video/out/opengl/context_android.c) isn't informed about the dimensions being changed
a terrible workaround can be found here: http://sprunge.us/VZUU?diff

**Comment by xyzz** [comment]
An easy and 100% way to reproduce this - go to AndroidManifest.xml and remove `android:screenOrientation="sensorLandscape"`. Start playing a video in landscape mode, then rotate your device into portrait mode.

**Comment by xyzz** [comment]
here's another workaround (the one in the first post doesn't work for me: vf del complains about invalid index)

```
    override fun surfaceChanged(holder: SurfaceHolder, format: Int, width: Int, height: Int) {
        MPVLib.setPropertyString("vid", "no")
        MPVLib.setPropertyInt("vid", 1)
    }
```

**Comment by sfan5** [comment]
mpv pull request: #5287
fix for mpv-android: 6a1594c3346faa7d5df060ba5e5d1eea189774eb
